marklogic-data-hub-central/ui/e2e/cypress/integration/explore/scenarios.queries.tsx

```diff
@@ -682,6 +683,7 @@ describe("manage queries modal scenarios on detail page", () => {
     detailPage.clickBackButton();
     cy.waitForAsyncRequest();
     browsePage.waitForSpinnerToDisappear();
+    cy.wait(1000);
```

Comment:
Instead of giving an implicit wait, can we use something like waitForTableToLoad() for waiting explicitly until the results are shown.

Reply:
Yes we use cy.waitForAsyncRequest(); line 684 for waiting until requests are resolved. However, it sometimes may not  work on DHS or page rendering takes extra time.

Reply:
Yes I agree we are waiting for the requests and also spinner to disappear . May be waiting for the table/card to load can fix this.